### ISSUE TITLE
Error listing 'hints' reader

### ISSUE THREAD

**Issue by johnnywalls** [nilp0inter/binlog#10]
Using binlog==5.1.0, with an initially empty database, after registering several readers, initially listing the readers for showing registry length works ok.  But after some time (without event entries), in this case possibly due to batch write timeout, a new 'hints' reader appears, and accessing its registry length fails with an unpack error:

```
error                                     Traceback (most recent call last)
<ipython-input-7-cd6e805b9a0e> in <module>()
----> 1 reader.registry.length()

/lib/python3.4/site-packages/binlog/registry.py in length(self)
     58     def length(self):
     59         return {'memory': len(self.memory.registry.acked),
---> 60                 'disk': len(self.db)}
     61 
     62     @property

/lib/python3.4/site-packages/binlog/util.py in wrapper(*args, **kwds)
     13         def wrapper(*args, **kwds):
     14             try:
---> 15                 return f(*args, **kwds)
     16             except self.exp as exc:
     17                 raise self.mask from exc

/lib/python3.4/site-packages/binlog/registry.py in __len__(self)
    120                     return 0
    121                 else:
--> 122                     return sum(1 for _ in cursor.iternext())
    123 
    124     def _get_segment_by_pos(self, pos):

/lib/python3.4/site-packages/binlog/registry.py in <genexpr>(.0)
    120                     return 0
    121                 else:
--> 122                     return sum(1 for _ in cursor.iternext())
    123 
    124     def _get_segment_by_pos(self, pos):

/lib/python3.4/site-packages/binlog/cursor.py in _iterate(self, iterator, keys, values)
    155         if keys and values:
    156             for raw_key, raw_value in iterator:
--> 157                 yield (self._from_key(raw_key),
    158                        self._from_value(raw_value))
    159         elif keys:

/lib/python3.4/site-packages/binlog/cursor.py in _from_key(self, data)
    106 
    107     def _from_key(self, data):
--> 108         return self.db.K.python_value(data)
    109 
    110     def _to_value(self, data):

/lib/python3.4/site-packages/binlog/serializer.py in python_value(value)
     12     @staticmethod
     13     def python_value(value):
---> 14         return struct.unpack("!Q", value)[0]
     15 
     16     @staticmethod

error: unpack requires a bytes object of length 8

```

Apparently, this issue could be resolved in commit https://github.com/nilp0inter/binlog/commit/ee42229b2e48e883991e8a723413c2a0a329df78, but it might be worth the verification.  Perhaps does capitalization matter (eg, the automatically created reader is 'hint' but not 'Hints')?

```
   ...:             with eventlog.reader(name) as reader:
   ...:                 print(name, reader)
   ...:             
...
hints <binlog.reader.Reader object at 0x7f724d07c898>
...

```

**Comment by nilp0inter** [comment]
This issue should be fixed in version 5.1.1, please check it and let me know.

**Comment by johnnywalls** [comment]
It's working correctly for our use cases, thank you very much, indeed!
